src/components/search/partials/search.html

```diff
@@ -11,10 +11,10 @@
     <i class="fa fa-search"></i>
     <button class="fa fa-remove-sign ga-icon ga-btn" ng-click="clearInput()"
        ng-mousedown="preClear($event)" ng-hide="query==''"></button>
-
     <span class="ga-search-dropdown"
           ng-show="restat.sets()"
-          ng-class="'ga-search-' + restat.sets()">
+          ng-class="'ga-search-' + restat.sets()"
+          ng-if="!(query==''||query=='undefined')">
```

Comment:
There is already a `ng-show`, you should use it: 
```
ng-show="query && restat.sets()"
```

Reply:
Ok thx a lot! I accordingly adapted the unittests. Test link is updated.